### DISCUSSION TITLE
Change fee receiver or build our own API/backend?

### DISCUSSION THREAD

**Discussion by wolfer21** [OneKeyHQ/app-monorepo#3440] [Q&A]
Hi there, is it possible/allowed to change the fee receiver for the swaps? if yes how can we change it? also is it possible to change the blockchain explorer from cn.etherscan.com to etherscan.io? and is there any way to gain access to the api endpoint source code or any indication on how to build our own backend/ API or what standard you used?

Thank you!

**Comment by irajsava** [comment]
# _****_
